os/arch/arm/src/imxrt/imxrt_adc.c

```diff
@@ -754,6 +764,26 @@ static struct imxrt_dev_s g_adcpriv = {
 
 static struct adc_dev_s g_adcdev;
 
+static inline void imxrt_adc_pins_init(void)
```

Comment:
I think we can remove inline from this function. It has 3 function call.
What is your opinion that?